doc/manual/source/configuration.rst

```diff
@@ -44,6 +44,18 @@ Note that details are provided for each unit and each target.
     # Where should the HTTP server listen on?
     http-listen = ["127.0.0.1:8080"]
 
+    # The proxy servers to use for outgoing HTTP requests.
```

Comment:
Should we stick a warning here saying this option only works with the `socks` feature flag for the build? I guess trying to use the option on a build without the feature results in a somewhat generic serde/clap error that does not provide much of a hint to the end-user.

Reply:
Good point. I think serde just ignores it, which is worse.